src/structures/Guild.js

```diff
@@ -300,7 +300,7 @@ class Guild extends Base {
    * @readonly
    */
   get afkChannel() {
-    return this.client.channels.get(this.afkChannelID);
+    return this.client.channels.get(this.afkChannelID) || null;
```

Comment:
this already returns null

Reply:
It does not.
`Map#get` returns undefined when the key couldn't be found in the map.

Reply:
Despite the lack of need to test, I verified that it returns undefined (without this PR), as expected from Collection#get.